service/src/test/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsServiceTest.java

```diff
@@ -608,6 +608,104 @@ void shouldAddNewHearing() {
             assertThat(caseData.getHearingDetails())
                 .containsExactly(hearing1, hearing2, newHearing);
         }
+
+        @Test
+        void shouldSetStartDateHearingFieldsWhenHearingStartDateIsInThePast() {
+            LocalDateTime hearingStartDate = LocalDateTime.of(2010, 3, 15, 20, 20);
```

Comment:
Few places in this file where we could use LocalDateTime.now() rather than manually creating a date.

Reply:
discussed